### PR TITLE
feat: Add Hero section

### PR DESCRIPTION
**Pull request by irsayvid**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Add Hero section with sponsors ticker.
linked issue: #20 
There is slight linting issue in "HeroSection.jsx" as the image  url exceeded the max number of characters

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by irsayvid** [comment]
@riteshsp2000 @CIPHERTron @abhibhaw 
Please review and let me know if any changes are needed

**Comment by riteshsp2000** [comment]
Hey @irsayvid before continuing ahead.... there are quite a few changes required here.

1. You are not using the container component that we created. Not using that will make the content stretch through out the page which we don't want for larger screens.
2. The content part in the first section is not centered, both vertically and horizontally. Please do check that.
3. There is quite a lot of spacing errors around the graphic. I would suggest to align the image to the right instead of the center.

**Comment by irsayvid** [comment]
Hey @riteshsp2000 I've made the changes mentioned in the previous comment. Can you check and let me know about further issues if any?

File: package.json [hunk 1]
```diff
@@ -1,62 +1,62 @@
 {
-  "name": "project-oregano",
-  "private": true,
```
Comment (abhibhaw) [inline package.json:3]: @CIPHERTron @riteshsp2000  can we have a workspace specific eslint/prettier config for `lf` instead of `crlf`  for consistency

Comment (riteshsp2000) [reply package.json:3]: sure... ill check

**Review by riteshsp2000 — APPROVED** [review]

**Review by CIPHERTron — APPROVED** [review]

**Comment by riteshsp2000** [comment]
@all-contributors add @irsayvid for Code

**Pull request merged**